Classes/BITChannel.m

```diff
@@ -304,6 +305,15 @@ - (void)enqueueTelemetryItem:(BITTelemetryData *)item {
   });
 }
 
+- (UIApplicationState)checkApplicationStateForApplication:(UIApplication *)application {
+  __block UIApplicationState state;
+  dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
Looks like this will not work. I guess it should be `dispatch_sync`

Reply:
you are right...it was too late last night ;)